lib/convert.js

```diff
@@ -92,6 +93,14 @@ var sdk = require('postman-collection'),
       }
       // We can't expose this option to the user
       util.options.schemaFaker = true;
+      // setting it to the utils object so that we can use it in schemaFaker function
+      if (typeof data === 'object') {
+        dataSize = ((Buffer.byteLength(JSON.stringify(data), 'utf8') / 1024) / 1024); // size of input file in Mbs
```

Comment:
repetition of code
the if/else should only get the string. The conversion to MB can happen in a common fashion